__tests__/app.test.js

```diff
@@ -53,3 +53,39 @@ describe("GET /api/topics", () => {
       });
   });
 });
+
+describe("GET /api/article/:article_id", () => {
+  test("200: Responds with an article object of a specific id with all the required properties", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body: { article } }) => {
+        expect(article).toEqual({
+          title: expect.any(String),
+          topic: expect.any(String),
+          author: expect.any(String),
+          body: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+          article_img_url: expect.any(String),
+          article_id: 1,
```

Comment:
Well done for keeping the test flexible but making sure we still are checking that we've got the right article back!